sdk/python/packages/flet-core/src/flet_core/banner.py

```diff
@@ -16,31 +16,33 @@ class Banner(Control):
     ```
     import flet as ft
 
+
     def main(page):
+        page.horizontal_alignment = ft.CrossAxisAlignment.CENTER
+
         def close_banner(e):
-            page.banner.open = False
-            page.update()
+            page.close(banner)
```

Comment:
we can, actually, add `banner.close()` method, can we?

Reply:
If we are doing this, the next question will be, what about `banner.open()`, then `date_picker.pick_time()`... :)